src/index.tsx

```diff
@@ -148,3 +112,89 @@ export async function init(config: IClientConfig): Promise<EppoClient> {
 export function getInstance(): EppoClient {
   return EppoReactNativeClient.instance;
 }
+
+export class EppoPrecomputedReactNativeClient extends EppoPrecomputedClient {
```

Comment:
missing a `getPrecomputedInstance` (or perhaps a more react-y `usePrecomputedInstance`) method

Reply:
Added `getPrecomputedInstance` to be consistent with the other SDKs, though I like the idea of `usePrecomputedInstance` which is more react-ish